htdocs/web_portal/controllers/downtime/add_downtime.php

```diff
@@ -158,62 +158,103 @@ function draw(\User $user = null) {
         die(json_encode(array('UTC', 0)));
     }
 
-    // URL Mapping
-    // If the user wants to add a downtime to a specific site, show only that site's SEs
-    else if(isset($_GET['site'])) {
-        $site = \Factory::getSiteService()->getSite($_GET['site']);
-        if(\Factory::getRoleActionAuthorisationService()->authoriseAction(\Action::EDIT_OBJECT, $site, $user)->getGrantAction() == FALSE){
-           throw new \Exception("You don't have permission over $site");
-        }
-        $ses = $site->getServices();
-        $params = array('ses' => $ses, 'nowUtc' => $nowUtcDateTime->format('H:i T'), 'selectAll' => true);
-        show_view("downtime/add_downtime.php", $params);
-        die();
+    // URL Mapping for `site` and `se`
+    elseif (isset($_GET['site']) && isset($_GET['se'])) {
```

Comment:
```suggestion
    // URL Mapping for `site` and `se`.
    // If the user wants to add a downtime to a specific site and SE,
    // show only that site's SEs.
    elseif (isset($_GET['site']) && isset($_GET['se'])) {
```

Reply:
I have updated the comments to be more readable